src/main/kotlin/com/sourcegraph/cody/agent/CodyAgent.kt

```diff
@@ -244,7 +246,6 @@ private constructor(
         Files.copy(binarySource, binaryTarget, StandardCopyOption.REPLACE_EXISTING)
         val binary = binaryTarget.toFile()
         if (binary.setExecutable(true)) {
-          binary.deleteOnExit()
```

Comment:
so did we delete the file here? how did the problems appear then?

Reply:
This only happens on Jim exit, not on the agent crash.